pegasus_wrapper/workflow.py

```diff
@@ -72,6 +72,8 @@ class WorkflowBuilder:
     # Path to the replica catalog to store checkpointed files
     _replica_catalog: Path = attrib(validator=instance_of(Path))
     _category_to_max_jobs: Dict[str, int] = attrib(factory=dict)
+    # Call include an experiment_name so that jobs are more identifiable on SAGA
+    _experiment_name: str = attrib(kw_only=True, default="")
```

Comment:
Why are we using this name instead of the workflow name? I think this should be included in the comment as it's not immediately obvious.

I have a guess, but to confirm: The workflow name isn't necessary (so unlikely to be specified) and the default is uninformative for what we want to do with it. On the other hand the experiment name will be specified and informative for most if not all workflows (by Vista convention).

Reply:
Yes, you are correct. I'm going off the convention set by Eric and Ryan, who often made a point to specify experiment names, but not workflow names. We could use workflow name is someone has a strong opinion one way or another. Whatever we go with should just be the convention from now on.

Reply:
Thanks for confirming. I think experiment name are the way to go here.

Reply:
Great, I'll add a comment and merge.